docs/plugins.rst

```diff
@@ -155,3 +155,37 @@ Usage
     const app = new VoxaApp({ Model, variables, views });
 
     plugins.autoLoad(app, { adapter });
+
+
+
+
+S3Persistence plugin
+--------------------
+
+It stores the user's session attributes in a file in an S3 bucket.
```

Comment:
add a comment that instructs developers to use this functionality with Alexa hosting service.

Reply:
I agree, you can add some recommendations on which situation this plugin is most useful.

Reply:
@rmberrios @wuelcas done!